apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/AplDbVersion.java

```diff
@@ -688,7 +688,7 @@ protected int update(int nextUpdate) {
             case 252:
                 // SHARDING meta-info inside main database
                 apply("CREATE TABLE IF NOT EXISTS shard (shard_id BIGINT NOT NULL, shard_hash VARBINARY, " +
-                        "shard_height INT not null default 0, shard_state BIGINT default 0, zip_hash_crc VARBINARY, generator_ids ARRAY )");
+                        "shard_height INT not null default 0, shard_state BIGINT default 0, zip_hash_crc VARBINARY, generator_ids ARRAY NOT NULL)");
```

Comment:
Behavior was changed, we should allow null here to save shard record during sharding process multiple times. ALready fixed in develop

Reply:
Reverted back